### PR TITLE
Notices Reduxification: missing docs

### PR DESCRIPTION
**Pull request by artpi**
In my notices reduxification efforts, I missed few doc items

CC @ebinnion

### CODE REVIEW

File: client/state/notices/README.md [hunk 1]
```diff
@@ -40,11 +40,24 @@ this.props.successNotice(
...
+* `warningNotice()`
 
 ## Options
 
 The first argument is the text to be displayed on the notice. The second argument is an optional object and accepts some properties:
 
-* `duration: 5000`: (optional) Duration in milliseconds to display the notice before dismissing.
-* `showDismiss: false`: (optional) To indicate if dismiss button should be rendered within the overlay.
+* `id`: ID for your notice
+* `duration: 5000`: (optional, default null) Duration in milliseconds to display the notice before dismissing.
```
Comment (gwwar) [inline client/state/notices/README.md]: Would it be possible to blend in the default information to the option description?

For example: 
```
`duration: 5000`: (optional) Time in milliseconds before notice removes itself from page. If duration is not specified, notices will stay on page forever.
```

Comment (artpi) [inline client/state/notices/README.md]: It would, but is there an upside?

Comment (ebinnion) [inline client/state/notices/README.md]: I'm not a fan of naming the options like this: `duration: 5000`. To me, that feels like `5000` is the default. 

Perhaps,

`duration: null`: (optional) – Time in milliseconds before notice removes itself from page. If duration is not specified, notices will stay on page forever.

Comment (gwwar) [inline client/state/notices/README.md]: When skimming a README, I wouldn't expect a default section, and would most likeli stop reading after this line.

Comment (gwwar) [inline client/state/notices/README.md]: > I'm not a fan of naming the options like this: duration: 5000. To me, that feels like 5000 is the default.

@ebinnion oh good eyes. The suggested format looks good. :+1:

Comment (artpi) [inline client/state/notices/README.md]: Thanks @ebinnion , @gwwar , I changed it according to your pointers :)

**Pull request merged**